### PR TITLE
fixing tableColumns

### PR DESCRIPTION
**Pull request by olhabn**
ticket: https://nanopay.atlassian.net/browse/NP-1741
nanopay pr: https://github.com/nanoPayinc/NANOPAY/pull/10295

### CODE REVIEW

File: src/foam/nanos/ticket/Ticket.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ foam.CLASS({
     'type',
     // REVIEW: view fails to display when owner in tableColumn, the 2nd entry in allColumns is undefined.
     // 'owner',
-    'createdBy',
+    'createdBy.id',
```
Comment (jlhughes) [inline src/foam/nanos/ticket/Ticket.js]:
```suggestion
    'createdBy.legalName',
```

File: src/foam/nanos/ticket/TicketComment.js [hunk 2]
```diff
@@ -27,8 +27,8 @@ foam.CLASS({
 
   tableColumns: [
     'created',
-    'createdBy',
-    'createdByAgent',
+    'createdBy.id',
```
Comment (jlhughes) [inline src/foam/nanos/ticket/TicketComment.js]:
```suggestion
    'createdBy.legalName',
```

File: src/foam/nanos/ticket/TicketComment.js [hunk 3]
```diff
@@ -27,8 +27,8 @@ foam.CLASS({
...
+    'createdByAgent.id',
```
Comment (jlhughes) [inline src/foam/nanos/ticket/TicketComment.js]:
```suggestion
    'createdByAgent.legalName',
```

File: src/foam/nanos/ruler/Rule.js [hunk 4]
```diff
@@ -35,12 +35,14 @@
...
     'name',
-    'ruleGroup',
+    'ruleGroup.id',
     'enabled',
     'priority',
     'daoKey',
-    'createdBy',
-    'lastModifiedBy'
+    'createdBy.firstName',
+    'createdBy.lastName',
```
Comment (jlhughes) [inline src/foam/nanos/ruler/Rule.js]:
```suggestion
```

File: src/foam/nanos/ruler/Rule.js [hunk 5]
```diff
@@ -35,12 +35,14 @@
...
     'id',
     'name',
-    'ruleGroup',
+    'ruleGroup.id',
     'enabled',
     'priority',
     'daoKey',
-    'createdBy',
-    'lastModifiedBy'
+    'createdBy.firstName',
```
Comment (jlhughes) [inline src/foam/nanos/ruler/Rule.js]:
```suggestion
    'createdBy.legalName',
```

File: src/foam/nanos/ruler/Rule.js [hunk 6]
```diff
@@ -35,12 +35,14 @@
...
+    'createdBy.lastName',
+    'lastModifiedBy.firstName',
+    'lastModifiedBy.lastName'
```
Comment (jlhughes) [inline src/foam/nanos/ruler/Rule.js]:
```suggestion
```

File: src/foam/nanos/ruler/Rule.js [hunk 7]
```diff
@@ -35,12 +35,14 @@
...
-    'ruleGroup',
+    'ruleGroup.id',
     'enabled',
     'priority',
     'daoKey',
-    'createdBy',
-    'lastModifiedBy'
+    'createdBy.firstName',
+    'createdBy.lastName',
+    'lastModifiedBy.firstName',
```
Comment (jlhughes) [inline src/foam/nanos/ruler/Rule.js]:
```suggestion
    'lastModifiedBy.legalName',
```

File: src/foam/nanos/ticket/Ticket.js (hunk 1 above)
Comment (olhabn) [reply src/foam/nanos/ticket/Ticket.js]: fixed

File: src/foam/nanos/ticket/TicketComment.js (hunk 2 above)
Comment (olhabn) [reply src/foam/nanos/ticket/TicketComment.js]: fixed

File: src/foam/nanos/ruler/Rule.js (hunk 7 above)
Comment (olhabn) [reply src/foam/nanos/ruler/Rule.js]: fixed

File: src/foam/nanos/ruler/Rule.js (hunk 6 above)
Comment (olhabn) [reply src/foam/nanos/ruler/Rule.js]: fixed

File: src/foam/nanos/ruler/Rule.js (hunk 5 above)
Comment (olhabn) [reply src/foam/nanos/ruler/Rule.js]: fixed

File: src/foam/nanos/ruler/Rule.js (hunk 4 above)
Comment (olhabn) [reply src/foam/nanos/ruler/Rule.js]: fixed

File: src/foam/nanos/ticket/TicketComment.js (hunk 3 above)
Comment (olhabn) [reply src/foam/nanos/ticket/TicketComment.js]: fixed

**Pull request merged**